src/components/Modals/Onboarding/Onboarding.vue

```diff
@@ -86,18 +86,15 @@ export default Vue.extend({
     return {
       // TODO: Get real college, major, and minor lists
```

Comment:
Is this comment irrelevant now? Can we delete it?

Reply:
@tcho6319 removed.